### PR TITLE
[Task] Removed the functionality to input metadata html tags in SEO and settings

### PR DESCRIPTION
**Pull request by aryaantony92**
## Additional info  

Should be merged only after merging https://github.com/pimcore/pimcore/pull/14396 to 10.6 and 11.x

### CODE REVIEW

File: models/Document/Page.php
Comment (brusch) [inline models/Document/Page.php]: The getter and the property `metaData` needs to be removed as well. 
Also we need to remove the `metaData` column from the `documents_page` table in `install.sql` and we need a migration for that.

**Review by brusch — CHANGES_REQUESTED** [review]

File: bundles/CoreBundle/src/Migrations/Version20230222174636.php
Comment (kingjia90) [inline bundles/CoreBundle/src/Migrations/Version20230222174636.php]: please add a check that is not already existing before altering

```suggestion
        $this->addSql('ALTER TABLE documents_page ADD COLUMN `metaData` TEXT AFTER `description`');
```

**Review by kingjia90 — CHANGES_REQUESTED** [review]
LGTM, waiting for the ALPHA release (where 10.6 is merged back into 11) before merging

File: bundles/AdminBundle/public/js/pimcore/document/pages/settings.js
Comment (kingjia90) [inline bundles/AdminBundle/public/js/pimcore/document/pages/settings.js:23]: There is a leftover here

https://github.com/pimcore/pimcore/blob/113e29bd934c3ed54be6666498c75f71ef4d6537/bundles/AdminBundle/public/js/pimcore/document/pages/settings.js#L119

Comment (aryaantony92) [reply bundles/AdminBundle/public/js/pimcore/document/pages/settings.js:23]: Done in [2f53525](https://github.com/pimcore/pimcore/pull/14411/commits/2f535259776aac8efdc2685d1aaeca404d3eb732)

**Review by kingjia90 — CHANGES_REQUESTED** [review]
Needs to check if it is still in use in tests, see https://github.com/pimcore/pimcore/actions/runs/4312212842/jobs/7522729102#step:11:3073

**Comment by aryaantony92** [comment]
> Needs to check if it is still in use in tests, see https://github.com/pimcore/pimcore/actions/runs/4312212842/jobs/7522729102#step:11:3073

I didn't find the usage in tests.

File: bundles/CoreBundle/src/Migrations/Version20230222174636.php
Comment (kingjia90) [inline bundles/CoreBundle/src/Migrations/Version20230222174636.php]:
```suggestion
        if(!$schema->getTable('documents_page')->hasColumn('metaData')) {
                $this->addSql('ALTER TABLE documents_page ADD COLUMN `metaData` TEXT AFTER `description`');
        }
```

**Comment by kingjia90** [comment]
please also remove `t('metadata')` from https://github.com/pimcore/pimcore/blob/fe80ccf4133ee3dcdbd395c3275628b9724acd48/bundles/AdminBundle/public/js/pimcore/document/pages/settings.js#L78

**Review by kingjia90 — CHANGES_REQUESTED** [review]
this metadata ~could be~ seems not related 
https://github.com/pimcore/pimcore/blob/fe80ccf4133ee3dcdbd395c3275628b9724acd48/bundles/AdminBundle/public/js/pimcore/settings/docTypes.js#L168-L169
under `Settings > Document Types`

**Comment by aryaantony92** [comment]
Done in [3ea494e](https://github.com/pimcore/pimcore/pull/14411/commits/3ea494e2ef27a408f55e1434c113d2d6e49d6447)

**Comment by aryaantony92** [comment]
> please also remove `t('metadata')` from
> 
> https://github.com/pimcore/pimcore/blob/fe80ccf4133ee3dcdbd395c3275628b9724acd48/bundles/AdminBundle/public/js/pimcore/document/pages/settings.js#L78

Done in [3ea494e](https://github.com/pimcore/pimcore/pull/14411/commits/3ea494e2ef27a408f55e1434c113d2d6e49d6447)

**Pull request merged**
